frontends/web/src/api/account.ts

```diff
@@ -65,6 +65,16 @@ export const getAccountsTotalBalance = (): Promise<ITotalBalance> => {
   return apiGet('accounts/total-balance');
 };
 
+type TETHAddressByAccountCode = {
+  success: boolean;
+  code: AccountCode;
+  name: string;
+}
```

Comment:
I assume you only get code and name if success is true, so you could hardcode to success: true or remove and do `type TETHAddressByAccountCode =  SuccessResponse & {`, this way TypeScript can error if success is false and you assume there is code or name.